addons/l10n_ua_psbo/data/account_chart_template_config.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="l10n_ua_psbo_chart_template" model="account.chart.template">
+            <field name="property_account_receivable_id" ref="ua_psbp_361"/>
+            <field name="property_account_payable_id" ref="ua_psbp_631"/>
+            <field name="property_account_expense_categ_id" ref="ua_psbp_901"/>
+            <field name="property_account_income_categ_id" ref="ua_psbp_701"/>
+            <field name="complete_tax_set" eval="True"/>
+            <field name="use_anglo_saxon" eval="True"/>
+            <field name="property_stock_account_input_categ_id" ref="ua_psbp_2812"/>
+            <field name="property_stock_account_output_categ_id" ref="ua_psbp_2811"/>
+            <field name="property_stock_valuation_account_id" ref="ua_psbp_281"/>
+            <field name="income_currency_exchange_account_id" ref="ua_psbp_711"/>
+            <field name="expense_currency_exchange_account_id" ref="ua_psbp_942"/>
+        </record>
+        <record id="base.UAH" model="res.currency">
+            <field name="active" eval="True"/>
```

Comment:
Maybe better to put it as the currency of the chart template?

Reply:
Yep, fixed this.